ERCS/erc-7579.md

```diff
@@ -0,0 +1,351 @@
+---
+eip: 7579
+title: Minimal Modular Smart Accounts
+description: Minimally required interfaces and behavior for modular smart accounts and modules
```

Comment:
There isn't enough here to distinguish this proposal from other (possibly hypothetical) smart account standards. What makes this proposal special? Being Minimal is fairly subjective.

Also try to include information in your description that isn't already contained in your title.